Modules/Nonunit/Review/test/itkImageFunctionTest.cxx

```diff
@@ -134,8 +134,8 @@ itkImageFunctionTest(int, char *[])
   image->SetRegions(region);
   image->Allocate();
 
-  auto origin = MakeFilled<ImageType::PointType>(0.0);
-  auto spacing = MakeFilled<ImageType::SpacingType>(1.0);
+  ImageType::PointType origin{};
+  auto                 spacing = itk::MakeFilled<ImageType::SpacingType>(1.0);
 
   image->SetOrigin(origin);
   image->SetSpacing(spacing);
```

Comment:
An `itk::Image` already has a zero-filled origin and a 1-filled spacing by default, by the way. Might as well just remove the `image->SetOrigin` and the `image->SetSpacing`...  Less = more  😺